src/components/Tooltip/index.tsx

```diff
@@ -1,13 +1,17 @@
 import { transparentize } from 'polished'
-import { ReactNode, useCallback, useEffect, useState } from 'react'
+import { ReactNode, useEffect, useState } from 'react'
 import styled from 'styled-components/macro'
 
 import Popover, { PopoverProps } from '../Popover'
 
+// TODO(lynnshaoyu): migrate noOps throughout web to a shared util file.
+const noOp = () => null
```

Comment:
`nop` or `noop` is more idiomatic

```suggestion
const noop = () => null
```

Reply:
done